pandas-stubs/core/arraylike.pyi

```diff
@@ -35,3 +40,6 @@ class OpsMixin:
     def __rdivmod__(self, other: Any) -> tuple[Self, Self]: ...
     def __pow__(self, other: Any) -> Self: ...
     def __rpow__(self, other: Any) -> Self: ...
+
+def dispatch_ufunc_with_out(self, ufunc: np.ufunc, method: str, *inputs, **kwargs): ...
+def dispatch_reduction_ufunc(self, ufunc: np.ufunc, method: str, *inputs, **kwargs): ...
```

Comment:
Those two functions don't seem to be documented

Reply:
I took the implementation of `DecimalArray` and those functions were used, so to make the typing work, I had to document them.  I think we have a lot of such functions in the stubs that are not documented, so we can just leave this one for now.

Reply:
up to you, wouldn't mind having an ignore comment in the test if we are not sure whether these functions are supposed to be public.

Reply:
> up to you, wouldn't mind having an ignore comment in the test if we are not sure whether these functions are supposed to be public.

OK, I did that.